providers/azureblob/src/test/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthenticationTest.java

```diff
@@ -74,6 +86,15 @@ void testIdempotent(HttpRequest request) {
       System.out.printf("%s: %d iterations before the timestamp updated %n", Thread.currentThread().getName(),
             iterations);
    }
+   
+   /**
+    * this test is similar to testIdempotent; it checks whether request is properly filtered when it comes to SAS Authentication
+    */
+   @Test(threadPoolSize = 3, dataProvider = "auth-sas-data", timeOut = 3000) 
```

Comment:
Remove everything but the data provider. Let's use the default config

Reply:
Done